torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -22,6 +24,31 @@ const std::string kClientActiveCalls = "agent.client_active_calls";
 const std::string kServerActiveCalls = "agent.server_active_calls";
 const std::string kServerActiveAsyncCalls = "agent.server_active_async_calls";
 
+std::string guessAddress(tensorpipe::transport::uv::Context& uvContext) {
```

Comment:
IIUC, this is just moving code from `startImpl` into its dedicated function?

Reply:
Yes, correct.